rpm/rygel.spec

```diff
@@ -1,25 +1,27 @@
 
 Name:          rygel
-Version:       0.40.1
+Version:       0.42.5
 Release:       1
 Summary:       A collection of UPnP/DLNA services
 
 License:       LGPLv2+
 URL:           https://wiki.gnome.org/Projects/Rygel
 Source0:       %{name}-%{version}.tar.xz
-Patch0:        0001-Completely-disable-doc-generation-because-we-dont-ha.patch
+Patch0:        0001-Completely-disable-doc-generation-because-we-don-t-h.patch
+Patch1:        0002-Remove-x11-dependency.patch
+Patch2:        0003-Use-simple-engine-by-default.patch
 
 BuildRequires: pkgconfig
 BuildRequires: meson
```

Comment:
Could use the opportunity to add ccache here as well

Reply:
As discussed on the glib2 PR, better not do that as it's not a real dependency, but meson just detecting the existence of ccache wrong. Possibly sb2 bug or something. Could fix the real bug or add a workaround dependency on one place.

Reply:
Do whatever you want. I'm just saying that the package does not build out of the box without ccache. We have been adding the BuildRequires here and there, whenever someone has stumbled on the issue. I don't see why rygel should be an exception.

Reply:
Well it's wrong and papers over issue elsewhere. That's silly. Ccache dependency should be removed from nearly everywhere.